spec/features/trains_show_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+feature 'User goes to the trains show page' do
+  scenario 'displays train details' do
+    Train.create(name: "ruby", description: "starts at ignition and ends with rails", founding_year: "2011-12-10")
+    visit "/trains/1"
+    expect(page).to have_content("ruby")
```

Comment:
If your show page is supposed to show the description and founding year too, please add tests for these!